### PR TITLE
Fixes to release candidate v6.0.0-rc.1

### PR DESCRIPTION
**Pull request by tiyash-basu-sociomantic**
The issues addressed in the commits here were identified while testing swarm `v6.0.0-rc.1` with dhtproto.

### CODE REVIEW

File: src/swarm/client/RequestSetup.d [hunk 1]
```diff
@@ -718,6 +718,24 @@ public template Suspendable ( )
...
+
+        Copies the value of the suspend unregister member into the provided
+        request params class instance.
+
+        Params:
+            params = IRequestParams instance to write into
+
+    ***************************************************************************/
+
+    private void setup_suspend_unregister ( IRequestParams params )
```
Comment (gavin-norman-sociomantic) [inline src/swarm/client/RequestSetup.d:732]: Was this removed by accident or missing?

Comment (tiyash-basu-sociomantic) [reply src/swarm/client/RequestSetup.d:732]: Turns out this was accidentally removed by me in https://github.com/sociomantic-tsunami/swarm/commit/6e26ffa.

Comment (tiyash-basu-sociomantic) [reply src/swarm/client/RequestSetup.d:732]: This method had a comment `// deprecated: remove in swarm v6`.
This means that the commit adding this in this PR should be removed, and dhtproto should be fixed instead?

Comment (gavin-norman-sociomantic) [reply src/swarm/client/RequestSetup.d:732]: Looks to me like it meant "remove the static if". (That's without looking into things more deeply.)

Comment (gavin-norman-sociomantic) [reply src/swarm/client/RequestSetup.d:732]: If it'd been me writing that deprecation comment (which it most likely was), I would have written it in the method doc if I'd meant to remove the whole method ;)

Comment (tiyash-basu-sociomantic) [reply src/swarm/client/RequestSetup.d:732]: The tests in dhtproto pass now, with the static if removed.

Comment (tiyash-basu-sociomantic) [reply src/swarm/client/RequestSetup.d:732]: For now, I have updated the commit description to better indicate the change.

**Pull request merged**